linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/Emitter.scala

```diff
@@ -515,7 +515,9 @@ object Emitter {
 
       // Implementation of Float_% and Double_%
       callStaticMethod(WasmRuntimeClass, fmodfMethodName),
-      callStaticMethod(WasmRuntimeClass, fmoddMethodName)
+      callStaticMethod(WasmRuntimeClass, fmoddMethodName),
+
+      callMethodStatically(JSRPCClass, handleMessageMethodName)
```

Comment:
I'm wondering how can I add symbol requirements only in a test environment? (and `genWasmEssentialFunctions` in `CoreWasmLib` 🤔 )

Reply:
Ah, that's a good idea 😄 
Do you mean like this? https://github.com/scala-wasm/scala-wasm/pull/59/commits/40b76ddc33f17f852a2e16146bad2f9fee828c04

Reply:
> inside the definition of init

Sounds nice! And with that way I could remove the length check in `handleMessage0` which is required because of dummy call.